### PR TITLE
Creates dockerfile

### PR DESCRIPTION
**Pull request by pcm32**
This PR adds a dockerfile which will be automatically built from our quay.io account for develop and tags. The container itself can be tried in the meantime on this URI: `quay.io/ebigxa/db-scxa-module:latest`. It also splits travis testing into container building a functionality testing.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.8
```
Comment (alfonsomunozpomer) [inline Dockerfile:1]: Why are you not using a PostgreSQL 10 image? Just curious...

Comment (alfonsomunozpomer) [reply Dockerfile:1]: Oh, your’re compiling R, never mind!

**Review by alfonsomunozpomer — APPROVED** [review]
Everything looks ok. I didn’t know about quay.io, though.

**Comment by pcm32** [comment]
Thanks for the review @alfonsomunozpomer! I should have set to WIP though before :-). Still hammering travis, found a missing package when installing via conda.

**Pull request merged**